lib/checks/aria/aria-errormessage-evaluate.js

```diff
@@ -24,10 +24,10 @@ import { isVisibleToScreenReaders } from '../../commons/dom';
  * @memberof checks
  * @return {Mixed} True if aria-errormessage references an existing element that uses a supported technique. Undefined if it does not reference an existing element. False otherwise.
  */
-function ariaErrormessageEvaluate(node, options, virtualNode) {
+export default function ariaErrormessageEvaluate(node, options, virtualNode) {
```

Comment:
What is the motivation behind this change?

Reply:
Follows our coding guidelines.

> If you encounter any code that we maintain that does not put the default export at the top, you should update the file to do so.

Reply:
What is the motivation behind putting the default export at the top?

Reply:
Ensures that when you open the file the main code path is the first thing you read.